starter_code/queries.md

```diff
@@ -2,82 +2,106 @@
 
 # Answers
 
-### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
+query:
+projection:
+sort:
+skip: 
+limit: 
 
-<!-- Your Code Goes Here -->
+### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
+query: { name: {$eq: 'Babelgum'} }
+projection: { name: 1 }
```

Comment:
Yup ! And the _id field always shows by default, if you do not want to have it, you have to say it explicitely in the projection : 
Projection: {name: 1, _id: 0}